components/dashboard/src/app/AppRoutes.tsx

```diff
@@ -218,8 +220,8 @@ export const AppRoutes = () => {
 
                     {repoConfigListAndDetail && (
                         <>
-                            <Route exact path="/repositories" component={RepositoryListPage} />
-                            <Route exact path="/repositories/:id" component={RepositoryDetailPage} />
+                            <Route exact path="/configurations" component={ConfigurationListPage} />
```

Comment:
Done this mostly because it's migrated from https://github.com/gitpod-io/gitpod/pull/18960, but we still need mutual agreement on this change.